docs/modules/commons/pages/vividus-steps.adoc

```diff
@@ -4,6 +4,31 @@
 
 Here one could find description of the steps that are delivered with Vividus itself without any plugins required.
 
+=== Initialize variable with value
+
+Initializes a variable with the specified value.
+
+[source,gherkin]
+----
+When I initialize $scopes variable `$variableName` with value `$variableValue`
```

Comment:
`When` or `Given` ?

Reply:
well, Given seems to be more logically correct